### PR TITLE
feat: Remove climate setting schedules

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
Closes https://github.com/seamapi/fake-seam-connect/issues/211

### CODE REVIEW

**Review by razor-x — APPROVED** [review]

**Comment by andrii-balitskyi** [comment]
@razor-x for some reason "Containerize" job fails. I don't think it's related to my changes. Do you have any ideas why it's failing?

**Comment by razor-x** [comment]
@andrii-balitskyi I ran npm update which fixed the build errors, but looks like there were some changes to the types that broke `create_multiple`. Those will need to be fixed in this PR as well.

File: package.json [hunk 1]
```diff
@@ -61,9 +61,7 @@
     "npm": ">= 9.0.0"
   },
   "optionalDependencies": {
-    "@seamapi/fake-devicedb": ">=1.0.0-rc.0",
-    "zustand": "^4.3.7",
-    "zustand-hoist": "^2.0.0"
```
Comment (razor-x) [inline package.json:66]: @andrii-balitskyi Why were these removed? Can we add them back?
